core/src/data_sources/data_source.rs

```diff
@@ -596,6 +596,7 @@ impl DataSource {
         Ok(())
     }
 
+    // What is this used for?
```

Comment:
Used to create the qdrant collection. Can now be removed entirely :+1:

Reply:
Removed.